setup.py

```diff
@@ -1,21 +1,11 @@
 #!/usr/bin/env python
-
-import os
-import sys
-
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
 
-# Our version ALWAYS matches the version of Django we support
-# If Django has a new release, we branch, tag, then update this setting after the tag.
-version = "3.1.13"
```

Comment:
We should probably make a last release with the old versioning before merging this